kbase-extension/static/kbase/js/widgets/narrative_core/jobCommChannel.js

```diff
@@ -289,104 +429,20 @@ define([
                             // If this job is not found in the incoming list of all
                             // jobs, then we must both delete it locally, and
                             // notify any interested parties.
-                            this.sendBusMessage(JOB, _jobId, 'job-deleted', {
+                            this.sendBusMessage(JOB, _jobId, 'job-status', {
```

Comment:
Couldn't find anywhere that this `job-deleted` message was used. Converting it to the more appropriate `job-status` `does_not_exist` message.